script/Arcueid/Day 13/13_00_ARC13_3.txt

```diff
@@ -213,42 +226,49 @@ I won't be able to miss Roa's 'death' no matter what.
 [sha:ca7a6a6a90c03d679fc0cb96b02fd5ae1d0514b8]{
 -- Page 249, Offset 32356.
 -- ここから先は一人で行ってください。わたしは別行動をしますから」
-From here on, you're on your own. I have other business to take care of."
+// Reading 別行動 as attacking from another angle, rather than "I need to go run some errands"
+From here on, you're on your own.
+ I'll be taking a different tack."
```

Comment:
hmm not sure about this expression. I think of "different tack" as being like "your way failed, lets try mine" rather than "an alternate approach". Dictionary seems to agree. I'm _assuming_ this is because Ciel's plan below is to just let arc and shiki fail but that's not really clear in this line and it makes it read a little awkwardly to me as a result. Without adding accidental foreshadowing I don't know that it can be made better.

a neutral "different approach" is probably fine either way. Maybe italicize different.